src/AppInstallerCommonCore/Runtime.cpp

```diff
@@ -309,6 +312,36 @@ namespace AppInstaller::Runtime
                 result = GetKnownFolderPath(FOLDERID_Profile);
                 create = false;
                 break;
+            case PathName::PortableAppUserRoot:
+                result = Settings::User().Get<Setting::PortableAppUserRoot>();
+                if (result.empty())
+                {
+                    result /= GetKnownFolderPath(FOLDERID_LocalAppData);
+                    result /= s_SecureSettings_Base;
+                    result /= s_DefaultPackagesDirectory;
+                }
+                create = true;
+                break;
+            case PathName::PortableAppMachineRootX64:
+                result = Settings::User().Get<Setting::PortableAppMachineRoot>();
+                if (result.empty())
+                {
+                    result /= GetKnownFolderPath(FOLDERID_ProgramFiles);

```

Comment:
```suggestion
                    result = GetKnownFolderPath(FOLDERID_ProgramFilesX64);
```
We should use the appropriate value here rather than just letting our process architecture drive it.

Reply:
Changed to `FOLDERID_ProgramFilesX64`